src/any_agent/config.py

```diff
@@ -98,7 +98,7 @@ class ServingConfig(BaseModel):
     endpoint: str = "/"
     """Will be pass as argument to `Starlette().add_route`"""
 
-    version: str = "0.1.0"
+    version: str = "1.0.0"
```

Comment:
I don't really understand what is this version for 😅

Reply:
Haha yeah I didn't either, but it's what was being used in the a2a-python demos, and I was trying to figure out why the server wasn't working even though it said it supported python 12 lol. Turns out this version number was not the answer, there's still an issue in a2a-python for py12 support. I'll undo this change because it's not needed